### PR TITLE
new update on insertion algo in c

### PR DESCRIPTION
**Pull request by gilbert-keter**
Added a README_insertion.md file for the insertion algorithm #108

### CODE REVIEW

File: Insertion_Sort.c
Comment (Abishethvarman) [inline Insertion_Sort.c:1]: make it in one single file .md

**Review by Abishethvarman — APPROVED** [review]

**Comment by Abishethvarman** [comment]
Rename the PR title in the correct way @GILBERTKETER

**Review by Thevapriyan07 — CHANGES_REQUESTED** [review]
Follow the previous PRs  and make it as a single file

**Comment by Abishethvarman** [comment]
@Thevapriyan07 do the changes possible and merge it. last day of hacktoberfest

**Review by Thevapriyan07 — APPROVED** [review]

**Pull request merged**